andela/nazgul-cfh
# UI-bug fixes

fixes(landing-signin-signup): modify landing, signin and signup page based on feedback

- landing page: reduce sizes of team images
- sign in, signup pages: change border and button focus to fit the color scheme.

#### What does this PR do?
- Fixes the teams profile images to fit size on mobile devices.
- Fixes sign in and signup form input borders, not to turn green on focus.
- Fixes buttons not to turn green on focus.

#### How should this be manually tested?
First cloning the application by running ```git clone https://github.com/andela/nazgul-cfh.git```.
Navigate to the nazgul-cfh directory.
Then running ```npm install``` to install all necessary dependencies.
finally running ```npm start``` to start the application.
Open your browser and navigate to ```localhost:3001``` for the landing page profile image implementation.
Navigate to ```localhost:3001/signin```. By filling the form it should be noticed that when clicking the sign-in button it no longer turns green.
Also same goes for the ```localhost:3001/signup``` page.

#### What are the relevant pivotal tracker stories?
- this is a link to the relevant PT story: https://www.pivotaltracker.com/story/show/156652086

#### Screenshots (if appropriate)
Input focus retains color scheme.
![image](https://user-images.githubusercontent.com/26353008/38586452-a87816aa-3d15-11e8-8cda-71dd7ba34545.png)

Social Icons are aligned on smaller screens.
![image](https://user-images.githubusercontent.com/26353008/38586469-b2fec268-3d15-11e8-83c2-e8f2e2ac38c1.png)

Teams profile images are smaller on small screens.
![image](https://user-images.githubusercontent.com/26353008/38586480-bbde92b4-3d15-11e8-8ae2-6fd4583091ae.png)

Tests passing 
![image](https://user-images.githubusercontent.com/26353008/38660992-3285ec0c-3e27-11e8-94ce-dfd06c867665.png)
